sdks/java/io/kafka/build.gradle

```diff
@@ -17,7 +17,7 @@
  */
 
 plugins { id 'org.apache.beam.module' }
-applyJavaNature(enableSpotbugs: false)
+applyJavaNature(enableSpotbugs: true)
```

Comment:
Please remove the variable the default is true so no need to do it twice.

Reply:
Thanks, fixed